py/selenium/webdriver/common/virtual_authenticator.py

```diff
@@ -0,0 +1,257 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import functools
+
+from base64 import urlsafe_b64encode, urlsafe_b64decode
+from enum import Enum
+from typing import Literal
```

Comment:
I just stumbled upon that line breaking Selenium with Python 3.7. `Literal` isn't available until [Python 3.8](https://docs.python.org/3/library/typing.html?highlight=typing#typing.Literal). In practice, `selenium` 4.1.4 is thus compatible with Python 3.8+.

Reply:
Is Selenium no longer supporting Python 3.7?

Reply:
@Hyask ; @risoms please pull latest `4.1.5` I have patched this bug

Reply:
Thank you for trying out the latest release and providing the feedback. Thank you @symonk for looking into it and helping patch it. Appreciate it!

Reply:
Thanks @symonk for the patch. And sorry for breaking things.